### DISCUSSION TITLE
Javers auditing on Spring JPA native queries

### DISCUSSION THREAD

**Discussion by vinaysuryawanshi26** [javers/javers#1221] [General]
Hello All,

We have scenario where we are using jdbc native query and native SQL through JPA repository but Javer is not able to identify change on such queries.

Is there any way or provision within javers that works on native SQL?

for instance
	@Transactional
    @Modifying(clearAutomatically = true)
    @Query(value = "**update Payment pmt set pmt.processInstanceId = :processInstanceId where pmt.paymentId = :paymentId**")
    **void updateProcessInstanceId(@Param("processInstanceId") String processInstanceId,@Param("paymentId") Long paymentId);**

**Comment by bartoszwalacik** [comment]
Hi, there is no such feature, Javers' diff algorithm works only on Java objects
